### PR TITLE
React park info

### PR DESCRIPTION
**Pull request by JesseChamberlain**
Created the ParkShow and ParkInfo elements in React with the API Fetch to our database.  All elements were created to meet Enzyme and Rspec tests.

### CODE REVIEW

File: app/views/parks/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Hello from parks show!</h1>
+
+<div id="app" data-id= <%= @id %> >
```
Comment (enthusiastick) [inline app/views/parks/show.html.erb:3]: I see what you did there.

File: react/src/containers/ParkShow.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      ]
+    }
+
+    return(
+      <div>
+        <h1>React ParkShow</h1>
+      	{this.state.park &&
+          <ParkInfo
+        	  ratings={ratings}
+        	/>
```
Comment (enthusiastick) [inline react/src/containers/ParkShow.js:45]: Careful with your formatting here, this doesn't look tidy.

File: spec/controllers/api/v1/parks_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      expect(park["avg_rating"]).to eq  5
+      expect(park["dog_friendly_avg_rating"]).to eq  5
+      expect(park["camping_avg_rating"]).to eq  5
+      expect(park["playground_avg_rating"]).to eq  5
+      expect(park["hiking_avg_rating"]).to eq  5
+      expect(park["scenery_avg_rating"]).to eq  5
+
+    end
+  end
+end
```
Comment (enthusiastick) [inline spec/controllers/api/v1/parks_controller_spec.rb:43]: I :heart: controller tests for API endpoints.

File: react/src/containers/ParkShow.js (hunk 2 above)
Comment (stenagli) [reply react/src/containers/ParkShow.js:45]: Got it from the React Docs: https://facebook.github.io/react/docs/conditional-rendering.html#inline-if-with-logical--operator

**Comment by enthusiastick** [comment]
Ship it.

File: react/src/containers/ParkShow.js (hunk 2 above)
Comment (enthusiastick) [reply react/src/containers/ParkShow.js:45]: @stenagli My complaint isn't with the conditional rendering, it's with the `ratings` prop and closing tag being indented too far on lines 44-45.

Comment (stenagli) [reply react/src/containers/ParkShow.js:45]: Ah I see. The indentation is normal on our computers, so we're not really sure why Github is changing it somehow.

**Pull request merged**